src/pages/Detail/DetailContents/ProductInfo/ProductInfo.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import useFetch from '../../../../hooks/useFetch';
+import BookmarkBorderIcon from '@mui/icons-material/BookmarkBorder';
+import BookmarkIcon from '@mui/icons-material/Bookmark';
+import * as S from './ProductInfo.style';
+import { useNavigate, useParams } from 'react-router-dom';
+import { API } from '../../../../config';
+
+const ProductInfo = ({ info }) => {
+  const params = useParams();
+  const navigate = useNavigate();
+  const [isUpdateWishListFlag, setIsUpdateWishListFlag] = useState(false);
+  let addedWishlist = false;
+
+  const {
+    author_name,
+    description,
+    height,
+    item_name,
+    length,
+    materials,
+    production_year,
+    weight,
+    width,
+  } = info;
+
+  //FIXME - mock data devUrl
+  //const getAllWishList = '/data/wishlistOne.json';
+
+  const getAllWishList = `${API.WISHLIST}`;
+
+  //const interlockToWishList = `${API.WISHLIST}`;
+  const { loading, data } = useFetch(getAllWishList);
+
+  if (!loading) {
+    data.data.map(data => {
+      if (data.item_id === Number(params.id)) {
+        addedWishlist = true;
+        return;
+      }
+    });
+  }
+
+  const addItemToWishList = id => {
+    const addUrl = `${getAllWishList}/items/${id}`;
+    const params = {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+    };
+    fetch(addUrl, params).then(res => {
+      if (res.ok) {
+        addedWishlist = !addedWishlist;
+        setIsUpdateWishListFlag(prev => !prev);
+      } else throw new Error(res);
+    });
+    navigate(`/detail/${id}`);
+  };
+
+  const removeItemToWishList = id => {
+    //백엔드 API 준비중
+    /*
+     * 작성될 코드
+     */
+    navigate(`/detail/${id}`);
+  };
+
+  const controlWishList = id => {
+    if (addedWishlist) {
+      //위시리스트 아이템 삭제 API 준비중
```

Comment:
```suggestion
      //TODO : 위시리스트 아이템 삭제 API 준비중
```
여기두요.

todo highlighter라는 extension도 있습니다.
전역에서 todo 라고 적혀있는 모든걸 하이라이트 해주는 extension이에요.

Reply:
앞서 위시리스트 제거 동작을 구현하면서 주석은 필요하지 않게 되었습니다.
말씀해주신 내용 (추후 작업해야할 내용 : TODO 주석 기재)은 확인하였으며, 차후부터 적용하도록 하겠습니다.